lib/models/api-key.ts

```diff
@@ -170,6 +171,56 @@ const getApiKeysModel = function (
 			});
 		},
 
+		/**
+		 * @summary Get all provisioning API keys by application (asc)
+		 * @name getProvisioningApiKeysByApplication
+		 * @public
+		 * @function
+		 * @memberof balena.models.apiKey
+		 *
+		 * @param {Object} [options={}] - extra pine options to use
+		 * @fulfil {Object[]} - apiKeys
+		 * @returns {Promise}
+		 *
+		 * @example
+		 * balena.models.apiKey.getProvisioningApiKeysByApplication().then(function(apiKeys) {
```

Comment:
Can we also get a test case, where we create a provisioning api key using `balena.models.application.generateApiKey()` and then retrieve it here?